sql/core/src/test/scala/org/apache/spark/sql/execution/joins/InnerJoinSuite.scala

```diff
@@ -198,7 +198,7 @@ class InnerJoinSuite extends SparkPlanTest with SharedSparkSession {
       }
     }
 
-    test(s"$testName using BroadcastNestedLoopJoin build left") {
+    testWithWholeStageCodegenOnAndOff(s"$testName using BroadcastNestedLoopJoin build left") { _ =>
```

Comment:
`BroadcastNestedLoopJoin` codegen is merged in `3.2` (current master), so keep test change here.